pkg/kvstore/etcd_debug.go

```diff
@@ -81,21 +82,26 @@ func EtcdDbg(ctx context.Context, cfgfile string, dialer EtcdDbgDialer, w io.Wri
 	iiw := iw.WithExtraIndent(3)
 	cfg.Context = ctx
 	cfg.Logger = zap.NewNop()
-	cfg.DialOptions = append(cfg.DialOptions, grpc.WithBlock(), grpc.WithContextDialer(dialer.DialContext))
-	cfg.DialTimeout = 1 * time.Second // The client hangs in case the connection fails, hence set a short timeout.
+	cfg.DialOptions = append(cfg.DialOptions, grpc.WithContextDialer(dialer.DialContext))
 
 	cl, err := client.New(*cfg)
 	if err != nil {
-		iiw.Println("❌ Failed to establish connection: %s", err)
+		iiw.Println("❌ failed to create gRPC client: %s", err)
 		return
 	}
 	defer cl.Close()
 
 	// Try to retrieve the heartbeat key, as a basic authorization check.
 	// It doesn't really matter whether the heartbeat key exists or not.
```

Comment:
Could you please update the comment to mention the fact that `client.New` does not block on connection failure, and hence we need to check the connection state to determine the type of failure?